public/app/features/inspector/InspectErrorTab.tsx

```diff
@@ -18,5 +18,20 @@ export const InspectErrorTab: React.FC<InspectErrorTabProps> = ({ error }) => {
       </>
     );
   }
-  return <div>{error.message}</div>;
+  if (error.message) {
+    try {
+      const [msg, json] = error.message.split(/\{(.+)/);
+      const jsonError = JSON.parse('{' + json);
+      const title = msg && msg !== '' ? <h3>{msg}</h3> : null;
+      return (
+        <>
+          {title}
+          <JSONFormatter json={jsonError} open={5} />
+        </>
+      );
+    } catch {
+      return <div>{error.message}</div>;
+    }
+  }
+  return <JSONFormatter json={error} open={2} />;
```

Comment:
⚠️  This change will format any error that ends with a valid json object, not only the ones from Azure, but I think I is an improvement to the current behaviour.

ℹ️  _I was trying out Google Cloud Monitoring to prove this point but turns out the error in that case is full of `\n` so not parsable json (sample: `query failed: {\n  \"error\": {\n    \"code\": 400,\n    \"message\": \"One or more errors p`) so my change does not improve Cloud Monitoring error formatting_